plutus-metatheory/src/Opts.hs

```diff
@@ -3,8 +3,6 @@
 
 module Opts where
 
-import Paths_plutus_metatheory (getDataDir)
```

Comment:
Can we remove `Paths_plutus_metatheory` from the cabal file now?

Reply:
Yes. done